scripts/setup-data-dir.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+datadir="${1}"
+
+if [[ -z "${1}" ]]; then
+    datadir="/var/lib/rkt"
+fi
+
+md() {
+    local dir="${1}"
+    local mode="${2}"
+
+    if [[ -e "${dir}" ]]; then
```

Comment:
I would use `-d` just to make sure that if an intruder file is there, we don't do `chown` on unknown files.